src/main/java/com/serotonin/bacnet4j/LocalDevice.java

```diff
@@ -405,8 +405,8 @@ public synchronized void terminate() {
         if (timer != null) {
             timer.shutdown();
             try {
-                if (!timer.awaitTermination(10, TimeUnit.SECONDS))
-                    LOG.warn("BACnet4J timer did not shutdown within 10 seconds");
+                if (!timer.awaitTermination(400, TimeUnit.SECONDS))
+                    LOG.warn("BACnet4J timer did not shutdown within 400 seconds");
```

Comment:
This is a very large increase in the timeout, which could effect other users. I think a parameter to the method would be better.